master/internal/job/job.go

```diff
@@ -2,111 +2,103 @@ package job
 
```

Comment:
nit: I feel like it might make sense to rename this package to jobservice, or alternatively we might want to rename Default to DefaultService. the authz stuff seems out of place tho cc @stoksc

Reply:
renaming job's global singleton to `DefaultService` in line with proxy/telemetry which all have their global singleton defined as `DefaultXYZ`, but the package doesn't have 'service' in the name.